src/components/Feeds/BusinessFeed.js

```diff
@@ -24,7 +26,17 @@ function BusinessFeed({ businesses, onSearch, selectedFilters, loadingState }) {
         selectedFilters={selectedFilters}
         isSearching={loadingState === LOADING_STATE.SEARCHING}
       />
-      {!initialLoad && hasResults && (
+      {(searching || initialLoad) && (
+        <Box mb={10}>
+          <SimpleGrid columns={[null, 1, 2]} spacing={10}>
+            {[...Array.from(new Array(4))].map((val, index) => (
```

Comment:
@Mhmdabed11 is there any chance we can up the array value here to 20? Happy to discuss if you think that's not ideal. I figure since 20 is the number of results that we show per-page it might be a nice transition!

My comment below may negate this so read on 😄

Reply:
Hello @magnificode , thank you for taking the time to check my pull request. Yes of course, there is no specific reason that I put 4, I just put it so it places a scroll on the page. But I think 20 makes more sense since, as you said, we are showing 20 results and it would be a nice transition.